projects/tpm2-tss/project.yaml

```diff
@@ -0,0 +1,6 @@
+homepage: "https://github.com/tpm2-software/tpm2-tss"
+primary_contact: "tadeusz.struk@intel.com"
```

Comment:
I recommend adding your email to `auto_ccs`. 
It will let you access testcases, oss-fuzz.com, and will notify when bugs are found or when the build is broken.